### DISCUSSION TITLE
Drive motors not driving on 2024 codebase

### DISCUSSION THREAD

**Discussion by tfield** [BroncBotz3481/YAGSL-Example#125] [Q&A] [answered]
After updating to WPILib 2024 (and then updating our rio and all of our SparkMAX controllers and our cancoder), I'm getting the control behaviour I want (so far) but my drive wheels are no longer moving! Why would my drive motors stop giving power, while my turn motors are fine?? https://youtube.com/shorts/PA5gG8RfHPI

Also, the drive motor alignment seems off too. What's going on and how do we fix it?

We also changed our absolute encoding offsets from degrees to rotations thinking that would help and it made NO noticeable impact, so perhaps things are dying sooner than this information is used.

**Comment by thenetworkgrinch** [comment]
I am actively investigating this, are you sure your conversion factors are correct?

**Comment by tfield** [comment]
Yes, we re-calculated them a few days ago and they worked really nicely on the 2023 codebase. TonyOn Jan 13, 2024, at 3:02 PM, T Grinch ***@***.***> wrote:﻿
I am actively investigating this, are you sure your conversion factors are correct?

**Comment by thenetworkgrinch** [comment, marked as answer]
I have just pushed the fix that Team 6238 had found.
